src/groups/bmq/bmqp/bmqp_ctrlmsg.xsd

```diff
@@ -1755,4 +1755,68 @@
       <element name='clusterNodeId'   type='int'    default='-1'/>
     </sequence>
   </complexType>
+
+  <!-- ========================================================================
+       ::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
+       :                      AUTHENTICATION MESSAGE                          :
+       #::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::#
+       ==================================================================== -->
+
+  <complexType name='AuthenticationMessage'>
+    <annotation>
+      <documentation>
+        This type is the top level type for any message being exchanged during
+        authentication of a connection with the broker, whether by a BlazingMQ
+        client or another broker.
+
+        choice.: enumerates all the different type of authentication packets
+
+        During negotiation, the process (whether a client using the libbmq SDK,
+        or a bmqbrkr) sends a 'authenticateRequest' message; to which the remote
+        peer will reply with a 'authenticateResponse' message.
+      </documentation>
+    </annotation>
+    <choice>
+      <element name='authenticateRequest'      type='tns:AuthenticateRequest'/>
+      <element name='authenticateResponse'     type='tns:AuthenticateResponse'/>
+    </choice>
+  </complexType>
+
+  <complexType name='AuthenticateRequest'>
+    <annotation>
+      <documentation>
+        This request is a messages sent from a client to a broker during
+        session initiation or reauthentication. The message indicates the client
+        is attempting to authenticate with the supplied authentication material
+        to initiate a session.
+
+        mechanism.: The authentication mechanism the client intends to use
+        data......: The client's authentication material
+      </documentation>
+    </annotation>
+    <sequence>
+      <element name='mechanism' type='string'/>
+      <element name='data'      type='hexBinary'  minOccurs='0'/>
+    </sequence>
+  </complexType>
+
+  <complexType name='AuthenticateResponse'>
+    <annotation>
+      <documentation>
+        Response of an 'AuthenticateRequest' request indicating the result of
+        the operation. The message is sent to a client from a broker during
+        authentication.
+
+        status.....: Status of the request
+        lifetimeMs.: The duration (in milliseconds) the client's session is
+                     valid for. The client must re-authenticate before the
+                     lifetime expires or it will be disconnected.
+      </documentation>
+    </annotation>
+    <sequence>
+      <element name='status'      type='tns:Status' />
+      <element name='lifetimeMs'  type='long'   minOccurs='0'/>
```

Comment:
What is the expected behavior when this value is not in the `AuthenticateResponse`?